cmd/flux/create_tenant_test.go

```diff
@@ -0,0 +1,49 @@
+package main
```

Comment:
In the `exportTenant` function replace `fmt.Println` with `rootCmd.Println`, this will make the golden test pass.

In the tests you need to set `--cluster-role=cluster-admin` as this flag gets restart for each test run.

```suggestion
//go:build e2e
// +build e2e

/*
Copyright 2025 The Flux authors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

package main
```